nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -511,6 +511,7 @@ class NetmikoDefault(DispatcherMixin):
 
     config_command = None  # This can be removed in future versions, as it is not used in the base class.
     offline_commands = False
+    bypass_commands = None
```

Comment:
Still wondering if we should just go ahead to bite the bullet and implement it similar to config_command/get_tcp_port etc.

somthing like

```py
    @classmethod
    def _get_bypass_commands(cls, obj) -> str:
        custom_field = obj.cf.get("bypass_commands")
        if isinstance(custom_field, int):
            return custom_field
        config_context = obj.get_config_context().get("bypass_commands")
        if isinstance(config_context, int):
            return config_context
        return cls.bypass_commands
```

Reply:
@itdependsnetworks I know we've agreed on this pattern, but not sure where we draw the line on adding them. If anything maybe we need to create a pattern that we also look for.

if 'nornir_overload_' in methodname then load them in as kwargs or something?

Reply:
I would prefer to wait for somebody to ask for it before writing code that may not get used.

Reply:
perhaps for this specific use case I'm okay waiting. But think we should still consider options going forward. Will let @itdependsnetworks wiegh in before merging.

Reply:
There is about a dozen other params we could use: https://ktbyers.github.io/netmiko/docs/netmiko/index.html#netmiko.BaseConnection.send_config_set
i don't think we should add them one at a time, and instead should pass along kwargs and let the user set this.

Reply:
Are we good doing that in a feature request via a new issue @itdependsnetworks , since this is a bug fix at the hard of this PR.

Reply:
> I wouldn't want to tbh, this is something we would need to support moving forward. Additionally, how sure are we that every user of Arista should have this option?
Do you mean that you wouldn't want to add the configcontext/customfield overrides for this feature in a separate request?

Reply:
> Do you mean that you wouldn't want to add the configcontext/customfield overrides for this feature in a separate request?

Correct. I don't think it makes sense to add known tech debt day 1.

Reply:
Talking with Jeff, this is what I am suggesting

```
    @classmethod
    def _get_netmiko_kwargs(cls, obj) -> str:
        custom_field = obj.cf.get("netmiko_kwargs")
        if isinstance(custom_field, int):
            return custom_field
        config_context = obj.get_config_context().get("netmiko_kwargs")
        if isinstance(config_context, int):
            return config_context
        return {}
```

Reply:
Putting here to avoid losing it. The thought is that nemiko doesn't allow for **kwargs on all methods. So we would have to do a bit of inspection based on what kwargs are passed via this cf/config context.  So something simple like below should be used before make sure the kwargs we're sending to each method call aren't going to result in `got an unexpected keyword argument '<>'` error.

```python
netmiko_kwargs = {"max_loops": 1, "use_ttp": True}
valid_params = inspect.signature(netmiko.BaseConnection.send_config_set).parameters

result_kwargs = {k: v for k, v in netmiko_kwargs.items() if k in valid_params}

```

Reply:
> Adding the suggested feature was not part of the planned scope of this bug fix. We can delay merging this PR until the feature is implemented but we'll need to add a new story to the backlog and the escalation will have to be put on hold until the feature can be implemented and properly tested.

There is no suggested feature being requested, there is a design that can/should be accepted.